frontend/src/components/BudgetLineItems/BLIReviewTable/BLIReviewRow.jsx

```diff
@@ -83,7 +89,7 @@ const BLIReviewRow = ({
             >
                 <input
                     className="usa-checkbox__input"
-                    id={budgetLine?.id}
+                    id={budgetLine?.id.toString()}
```

Comment:
Curious why we're suddenly converting this id to a string? Should we do this for all React ids? I'm not stopping up the review for it, I stopped it for the incumbent suggestion above. Just curious for my own edification.

Reply:
the red squiggles told me that html input ids can't be `number` but `string` 🙃